test/attr/attr_dynamic_member_lookup.swift

```diff
@@ -600,8 +600,8 @@ func keypath_with_subscripts(_ arr: SubscriptLens<[Int]>,
 
 func keypath_with_incorrect_return_type(_ arr: Lens<Array<Int>>) {
   for idx in 0..<arr.count {
-    // expected-error@-1 {{operator function '..<' requires that 'Lens<Int>' conform to 'Strideable'}}
-    // expected-error@-2 {{operator function '..<' requires that 'Lens<Int>.Stride' conform to 'SignedInteger'}}
+    // expected-error@-1 {{protocol 'Sequence' requires that 'Lens<Int>' conform to 'Strideable'}}
+    // expected-error@-2 {{protocol 'Sequence' requires that 'Lens<Int>.Stride' conform to 'SignedInteger'}}
```

Comment:
This is technically not accurate, the requirement is not coming from the Sequence protocol itself.

Reply:
Diagnostics here aren't great because we still don't have a full coverage, but that's something on my radar.